src/app/core/services/edit/edit.service.ts

```diff
@@ -463,6 +465,15 @@ export class EditService {
     return Promise.all(promises)
       .then((results) => {
         this.dataService.hideItemsInCurrentFolder(items);
+        this.analytics.notifyObservers({
+          entity: 'record',
+          action: 'move',
+          entityId: this.accountService.getAccount()?.accountId.toString(),
+          version: 1,
+          body: {
+            noTransmit: true,
```

Comment:
This pattern, where we send an "analytics" event that doesn't actually go to any analytics repository in order to let the checklist know that it should refresh, doesn't seem quite right to me. This might just be a naming issue; an events service that sends a subset of its events to analytics seems reasonable to me. But does it makes sense for the client to have an internal event saying we tried to move a record and a separate event fired on the back end saying we actually did move the record? Maybe, I'm not sure. Perhaps we should discuss this in sprint planning on Monday.